native-static-images/README.md

```diff
@@ -0,0 +1,111 @@
+# Building a Native Executable as Statically Linked or Mostly-Statically Linked.
```

Comment:
Update the title: 

Building Statically Linked or Mostly-Statically Linked Native Executables